src/domain/event/eventPreviewCard/EventPreviewCard.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { IconLocation } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import { EventFieldsFragment } from '../../../generated/graphql';
+import useLocale from '../../../hooks/useLocale';
+import IconClock from '../../../icons/IconClock';
+import getLocalisedString from '../../../utils/getLocalizedString';
+import PlaceText from '../../place/placeText/PlaceText';
+import EventKeywords from '../eventKeywords/EventKeywords';
+import { getEventPlaceholderImage, getEventStartTimeStr } from '../utils';
+import styles from './eventPreviewCard.module.scss';
+
+interface Props {
+  event: EventFieldsFragment;
+  link: string;
+}
+
+const EventCard: React.FC<Props> = ({ event, link }) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+
+  const id = event.id;
+  const name = getLocalisedString(event.name, locale);
+  const description = getLocalisedString(event.description || {}, locale);
+  const image = event.images[0]?.url;
+  const time = getEventStartTimeStr(event, locale, t);
```

Comment:
You could use getEventFields helper function to get these values

Reply:
goot catch, fixed :)